//user.proto主要规定了rpc调用的协议
syntax="proto3";

package fixbug;

option cc_generic_services=true;

message ResultCode
{
    int32 errcode=1;
    bytes errmsg=2;
}

//注册方法
message GetFriendsListRequest
{
    uint32 userid=1;
}

message GetFriendsListResponse
{
    ResultCode result=1;
    repeated bytes friends=2;
}

//好友模块
service FriendServiceRpc
{
    rpc GetFriendsList(GetFriendsListRequest) returns(GetFriendsListResponse);//rpc方法 定义远程调用传入的参数和返回类型
    
}